### PR TITLE
A lot has changed since then, I'm a new man

### PR DESCRIPTION
**Pull request by snydecor000**
The robot works as is, merging to master

### CODE REVIEW

File: src/org/redalert1741/steamworks/Robot.java [hunk 1]
```diff
@@ -93,28 +97,21 @@ public void robotInit()
 		driveAimer.setInputRange(-24,24);
 		driveAimer.setOutputRange(-.3,.3);
 		driveAimer.setAbsoluteTolerance(.5);
+		climber = new Climber(0, 1);
 		ReloadConfig();
 	}
-
+//========================================================================================================
```
Comment (randomtestfive) [inline src/org/redalert1741/steamworks/Robot.java:103]: this line is garbage

File: src/org/redalert1741/steamworks/Robot.java [hunk 2]
```diff
@@ -124,24 +121,24 @@ public void autonomousPeriodic()
 			drive.swerveAbsolute(0, -.001, 0, 0, false);
 		}
 	}
-
+//========================================================================================================
```
Comment (randomtestfive) [inline src/org/redalert1741/steamworks/Robot.java:124]: this one also

File: src/org/redalert1741/steamworks/Robot.java [hunk 3]
```diff
@@ -124,24 +121,24 @@ public void autonomousPeriodic()
...
 	@Override
     public void teleopInit()
-    { StartLogging("teleop",logger)
-    ; logger.addLoggable(navx)
-    ; logger.addLoggable(drive)
-    ; logger.writeAttributes()
-    ; ReloadConfig()
-    ; timer.reset()
-    ; timer.start()
+    { setupPeriodic("teleop")
```
Comment (randomtestfive) [inline src/org/redalert1741/steamworks/Robot.java:127]: -1 not enough weird formatting

File: src/org/redalert1741/steamworks/Robot.java [hunk 4]
```diff
@@ -157,30 +154,29 @@ public void teleopPeriodic()
...
-    		ReloadConfig();
+    		climber.climb(driver.getTriggerAxis(Hand.kRight));
+    	}
+    	else
+    	{
+    		climber.climb(0);
     	}
 	}
-
+//========================================================================================================
```
Comment (randomtestfive) [inline src/org/redalert1741/steamworks/Robot.java:169]: more garbage

File: src/org/redalert1741/steamworks/Robot.java [hunk 5]
```diff
@@ -196,8 +192,17 @@ public void testPeriodic()
     		}
     	}
 	}
+//========================================================================================================
```
Comment (randomtestfive) [inline src/org/redalert1741/steamworks/Robot.java:195]: stop

File: src/org/redalert1741/steamworks/SwerveModule.java [hunk 6]
```diff
@@ -29,7 +29,7 @@ public SwerveModule(CANTalon d, CANTalon a, AnalogInput e, String name)
 		SteerP = Config.getSetting("steerP",2);
 		SteerI = Config.getSetting("steerI",0);
 		SteerD = Config.getSetting("steerD",0);
-		SteerTolerance = Config.getSetting("Steering%Tolerance", .25);
```
Comment (randomtestfive) [inline src/org/redalert1741/steamworks/SwerveModule.java:32]: 🥇

File: src/org/redalert1741/steamworks/Manipulation.java [hunk 7]
```diff
@@ -3,12 +3,12 @@
...
 import com.ctre.CANTalon.TalonControlMode;
 
-public class Collection 
+public class Manipulation 
 {
 	CANTalon input;
 	CANTalon output;
 	
-	public Collection(CANTalon in,CANTalon out)
+	public Manipulation(CANTalon in,CANTalon out)
```
Comment (randomtestfive) [inline src/org/redalert1741/steamworks/Manipulation.java:11]: this is the best change

**Review by randomtestfive — CHANGES_REQUESTED** [review]
do better

File: src/org/redalert1741/steamworks/Robot.java (hunk 1 above)
Comment (snydecor000) [reply src/org/redalert1741/steamworks/Robot.java:103]: but why

File: src/org/redalert1741/steamworks/Manipulation.java (hunk 7 above)
Comment (snydecor000) [reply src/org/redalert1741/steamworks/Manipulation.java:11]: I wonder whose to blame for that one

**Review by randomtestfive — APPROVED** [review]

**Comment by randomtestfive** [comment]
wow it's so much better now

**Pull request merged**